tests/Core/PullToRefresh/FluentPullToRefreshTests.razor

```diff
@@ -0,0 +1,75 @@
+﻿@using Xunit;
+@inherits TestContext
+@code
+{
+    public FluentPullToRefreshTests()
```

Comment:
Your code coverage is only 31%. Could you add more Unit Tests to validate `ReleaseTemplate`, `LoadingTemplate`, `NoDataTemplate` , `CompletedTemplate`.

And `OnTouchStartAsync`, `OnTouchMoveAsync`, ... using code like this.

Example
```CSharp
// Arrange && Act
var cut = Render(@<FluentPullToRefresh Direction="@PullDirection.Down">Content that can be refreshed</FluentPullToRefresh>);

var div = cut.Find("div");
var touchArgs = new TouchEventArgs() { TargetTouches = new[] { new TouchPoint() { ClientY = 10 } } };
div.TriggerEvent("ontouchstart", touchArgs);

// Assert
// ...
```